### PR TITLE
Updates to docs for Operator 4.5.4 and Console v0.21.0

### LINKED ISSUE #569: [RELEASE] Operator 4.5.0 changes
Operator release [4.5.0](https://github.com/minio/operator/releases/tag/v4.5.0) has changes that impact the docs.

- Adds new `--interactive` flag to `kubectl minio tenant create` ([PR #1249](https://github.com/minio/operator/pull/1249))
- Can pass multiple client certificates ([PR #1263](https://github.com/minio/operator/pull/1263/files))

### LINKED ISSUE #590: [RELEASE] Console Release v0.20.5 changes for docs
Console release [v0.20.5](https://github.com/minio/console/releases/tag/v0.20.5) has changes that impact the docs:

- [x] You can now add environment variables in a Configuration tab in the Console when setting up tenants ([PR #2331](https://github.com/minio/console/pull/2331))
- [x] Policies have moved in the console to being under Identity ([PR #2348](https://github.com/minio/console/pull/2348))

### LINKED ISSUE #615: [RELEASE] Console version v0.21.0
[Console release 0.21.0 ](https://github.com/minio/console/releases/tag/v0.21.0) has changes that impact the docs:

- [ ] The Monitoring > Metrics view now has shows info on server metrics ([PR #2340](https://github.com/minio/console/pull/2340))
- [ ] Users can manually modify the YAML file on the Encryption page when creating a tenant ([PR #2354](https://github.com/minio/console/pull/2354))
- [ ] The Tiers view now hows the status of each tier ([PR #2291](https://github.com/minio/console/pull/2291))
- [ ] Site replication now supports separate accesskey/secretkey for different sites ([PR #2350](https://github.com/minio/console/pull/2350))

### LINKED ISSUE #634: [release] Operator v4.5.4 docs updates
Operator v4.5.4 has changes that impact the docs.

- [ ] Adds ability to set a pool name when creating or expanding a tenant ([PR #1322](https://github.com/minio/operator/pull/1322))

### PR DESCRIPTION
**Pull request by djwfyi**
# Updates for Operator 4.5.0

- Add --interactive flag info to kubectl create tenant docs
- Adds flag to kubectl minio tenant create doc
- Softens wording on bucket limit to be recommended in the MinIO console doc for PR#649
    
# Updates for Console v0.20.5
 
- Adds environment variable option to create tenant
- Creates Audit Log section to create tenant page and moves some options there

# Updates for console v0.21.0
    
- Adding info about the tiers status column when in the console
- Reworking site replication section for new changes to the layout in Console
- Skips documenting support for editing raw YAML for encryption page when creating a tenant, as that seems to not be supported any longer

# Updates for operator 4.5.4
    
- Adds --pool flag to kubectl minio tenant create
- Adds --pool flag to kubectl minio tenant expand
    
Closes #634
Closes #615
Closes #590
Closes #569

### CODE REVIEW

File: source/operations/install-deploy-manage/deploy-minio-tenant.rst [hunk 1]
```diff
@@ -485,10 +482,45 @@ Enabling SSE also creates :minio-git:`MinIO Key Encryption Service <kes>` pods i
      - Configure `Azure Key Vault <https://azure.microsoft.com/en-us/services/key-vault/#product-overview>`__  as the external KMS for storing root encryption keys. 
        See :ref:`minio-sse-azure` for guidance on the displayed fields.       
 
+.. _minio-tenant-audit-logging-settings:
+
+9) August Log Settings
```
Comment (ravindk89) [inline source/operations/install-deploy-manage/deploy-minio-tenant.rst:487]:
```suggestion
9) Audit Log Settings
```

File: source/operations/install-deploy-manage/deploy-minio-tenant.rst [hunk 2]
```diff
@@ -485,10 +482,45 @@ Enabling SSE also creates :minio-git:`MinIO Key Encryption Service <kes>` pods i
...
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+   :width: 100%
+
+   * - Field
+     - Description
+
+   * - Log Search STorage Class
```
Comment (ravindk89) [inline source/operations/install-deploy-manage/deploy-minio-tenant.rst:498]:
```suggestion
   * - Log Search Storage Class
```

File: source/reference/kubectl-minio-plugin/kubectl-minio-tenant-create.rst [hunk 3]
```diff
@@ -89,6 +91,23 @@ Flags
 
 The command supports the following flags:
 
+.. mc-cmd:: --interactive
+   :optional:
+
+   Offers command line prompts to request the information required to set up a new tenant.
+   Use this instead of the remaining flags when creating a new tenant.
```
Comment (ravindk89) [inline source/reference/kubectl-minio-plugin/kubectl-minio-tenant-create.rst:98]: "Mutually exclusive with all other arguments to this command" ?

**Review by ravindk89 — APPROVED** [review]
LGTM mod nits

File: source/operations/install-deploy-manage/deploy-minio-tenant.rst (hunk 1 above)
Comment (djwfyi) [reply source/operations/install-deploy-manage/deploy-minio-tenant.rst:487]: 👓

File: source/reference/kubectl-minio-plugin/kubectl-minio-tenant-create.rst (hunk 3 above)
Comment (djwfyi) [reply source/reference/kubectl-minio-plugin/kubectl-minio-tenant-create.rst:98]: Yeah, that says it better and more consistent with our language elsewhere.

**Pull request merged**